templates/arm-vnet-template.json

```diff
@@ -30,7 +30,12 @@
       "defaultValue": "aks-appgw"
     },
     "appgwSubnetCIDR": {
-      "type": "string"
+      "type": "string",
+      "defaultValue": "10.10.2.0/28"
```

Comment:
what is this? we probably shouldn't have a default unless this is to not have a breaking change temporarily

Reply:
it was just to make it easier to test - can remove if nobody wants it there?

Reply:
hmm, I see, still not really sure we should be letting defaults in =/

Reply:
can remove them all separately later, it shouldn't cause any harm since the rest of them have defaults

Reply:
yeah we either remove all of them or keep all of them.  It's easy enough to have a parameters file to test with